gui/control_main.py

```diff
@@ -2693,13 +2702,24 @@ def moveOmegaCB(self):
             {"relative": False}
         )
 
-    def moveEnergyCB(self):
+    def moveEnergy10eVCB(self):
```

Comment:
Maybe `moveEnergySmallCB(self)`? The threshold could be adjusted later or be implemented with a different value for other beamline(s) without having a bad name.

Reply:
Changed to a more meaningful name in https://github.com/NSLS-II/lsdc/pull/369/commits/2878abc67904a11851f1a4292848a1ab064cfc6a